examples/all-clusters-app/all-clusters-common/gen/IMClusterCommandHandler.h

```diff
@@ -0,0 +1,279 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+// THIS FILE IS GENERATED BY ZAP
+
+#include <stdint.h>
+
+#include "af-structs.h"
+#include "call-command-handler.h"
+#include "callback.h"
+#include "cluster-id.h"
+#include "command-id.h"
+#include "util.h"
+
+#include <app/Command.h>
+#include <core/CHIPTLV.h>
+
+// The IM_HAVE_XXX_CLUSTER_YYY_COMMAND is used for compatibility handlers.
```

Comment:
is there a TODO to remove them? They look odd inside namespaces (stutter and defines have no namespace scoping)

Reply:
Added a TODO